WcaOnRails/app/views/delegate_reports/edit.html.erb

```diff
@@ -11,7 +11,8 @@
     <% if @delegate_report.posted? %>
       <%= f.input :discussion_url %>
     <% end %>
-    <%= f.input :schedule_url %>
+    <% default_schedule_url = @competition.competition_venues.any? ? link_to_competition_schedule_tab(@competition) : '' %>
```

Comment:
Can you introduce a `has_schedule?` helper method to our competitions model? I don't think this code should be aware of the that that we store schedules as competition_venues.

Reply:
It [exists](https://github.com/thewca/worldcubeassociation.org/blob/a19005dce0cacd1225273f8b443b36fd2c689651/WcaOnRails/app/models/competition.rb#L710) ;)

Reply:
Thank you, that is much cleaner. I am now using that helper method :)